src/bin/install-plugins.php

```diff
@@ -63,8 +63,12 @@
 		
 		echo "Error: Plugin activation failed for {$plugin}:" . PHP_EOL;
 		
-		foreach ( $result->get_error_messages() as $message ) {
-			echo "- {$message}" . PHP_EOL;
+		foreach ($result->get_error_codes() as $code) {
+			echo "- " . $result->get_error_message( $code ) . PHP_EOL;
+			$data = $result->get_error_data( $code );
+			if ( ! empty( $data ) && is_string( $data ) ) {
+				echo "Data: " . $data;
```

Comment:
Thanks for the PR @Tofandel ! Sorry for the delay in reviewing it. 

This looks good, except maybe there needs to be a new line output after the data. What do you think?

```suggestion
				echo "Data: " . $data . PHP_EOL;
```